### DISCUSSION TITLE
Is Annotations Framework still in preview (1 year after announcement) ?

### DISCUSSION THREAD

**Discussion by petarrepac** [aws/aws-lambda-dotnet#1508] [General]
[https://aws.amazon.com/blogs/developer/introducing-net-annotations-lambda-framework-preview/](https://aws.amazon.com/blogs/developer/introducing-net-annotations-lambda-framework-preview/)
was published 13 APR 2022.

I wanted to rewrite a subsystem in Annotations, but now any expectation when it will be supported feels very risky.
The progress feels very slow.
We cannot use the pre-release version.

**Comment by normj** [comment]
We just released a new version today with several fixes to Annotations. 
![image](https://user-images.githubusercontent.com/1653751/236880382-a6b75b7f-0fd5-4a7f-a023-bf39cfb72837.png)

We gave this a long preview cycle to make sure we had the feedback this was a valuable project. We believe we have that feedback now and are moving forward with GA. We hope to do that soon and are working through the internal steps to make it GA (legal review and such).

Other then the library still being pre-release is there anything else you need from this library to use in production code? I get not wanting to use this in production if it is still pre-release.

**Reply by petarrepac** [reply]
This is great news!
Thank you and everyone involved.
Being in pre-release was the main blocker.
